src/User32.Desktop/User32.cs

```diff
@@ -1400,7 +1932,7 @@ public static partial class User32
         ///     <para>
         ///         If hWnd is -1, PeekMessage retrieves only messages on the current thread's message queue whose hwnd value is
         ///         NULL, that is, thread messages as posted by <see cref="PostMessage(IntPtr, WindowMessage, void*, void*)" />
-        ///         (when the hWnd parameter is <see cref="IntPtr.Zero" />) or <see cref="PostThreadMessage" />.
+        ///         (when the hWnd parameter is <see cref="IntPtr.Zero" />) or PostThreadMessage.
```

Comment:
Instead of removing the `<see cref` around `PostThreadMessage`, you can fix the build warning you get when you added a second overload by specifying the signature, for example:
`<see cref="PostThreadMessage(int, WindowMessage, IntPtr, IntPtr)" />`